Apps/W1/DynamicsGPHistoricalData/app/src/Pages/HistReceivablesDocument.Page.al

```diff
@@ -185,6 +185,34 @@ page 41006 "Hist. Receivables Document"
         }
     }
 
+    actions
+    {
+        area(Promoted)
+        {
+            actionref(ViewDistributions_Promoted; ViewDistributions)
+            {
+            }
+        }
+        area(Processing)
+        {
+            action(ViewDistributions)
+            {
+                ApplicationArea = All;
+                Caption = 'View Distributions';
+                ToolTip = 'View the G/L account distributions related to this transaction.';
+                Image = RelatedInformation;
+
+                trigger OnAction()
+                var
+                    HistGenJournalLines: Page "Hist. Gen. Journal Lines";
+                begin
+                    HistGenJournalLines.SetFilterOriginatingTrxSourceNo(Rec."Audit Code");
+                    HistGenJournalLines.RunModal();
```

Comment:
Minor comment here - does it need to be RunModal?
RunModal will block any other code from running, so the calling thread will wait for the dialog to close. This is needed if the page is doing the changes to the data that is shown in the main page or if we want to run some code afterwards so we need to wait. From this code it does not seem to be the case, so all calls could be changed to .Run().

If we need to wait, won't fix this comment.

Reply:
Will update to .Run(), thanks.